build_tools/generate_authors_table.py

```diff
@@ -196,9 +196,7 @@ def generate_table(contributors):
 
 
 def generate_list(contributors):
-    lines = []
-    for contributor in contributors:
-        lines.append("- %s" % (contributor["name"],))
+    lines = ("- %s" % (contributor["name"],) for contributor in contributors)
```

Comment:
could be a f-string

Reply:
True, but that's orthogonal to this merge request that applies _refurb_ suggestions. Perhaps it's better left to _pyupgrade_, which should fix automatically in a different merge request.

Reply:
We don't use `refurb` and it's not a tool we trust blindly. If we change a line of code, we should think about what we're doing, and here since it's being touched anyway, making it a f-string makes sense.